apps/editing-toolkit/editing-toolkit-plugin/wpcom-block-editor-nux/src/store.js

```diff
@@ -141,12 +141,12 @@ export const selectors = {
 	getShouldShowFirstPostPublishedModal: ( state ) => state.shouldShowFirstPostPublishedModal,
 };
 
-export function register() {
-	return registerStore( 'automattic/wpcom-welcome-guide', {
-		reducer,
-		actions,
-		selectors,
-		controls,
-		persist: true,
-	} );
-}
+export const store = createReduxStore( 'automattic/wpcom-welcome-guide', {
+	reducer,
+	actions,
+	selectors,
+	controls,
+	persist: true, // TODO @noahtallen: persist plugin probably doesn't work here.
```

Comment:
We do persist store data. This is it what the store looks like without the ETK patch.

<img width="876" alt="CleanShot 2023-03-15 at 21 06 09@2x" src="https://user-images.githubusercontent.com/1500769/225246553-1ace4f21-d5a0-4f86-853c-35ab4940cb75.png">

And we do rely on it for caching so we're not always fetching the "should they see the welcome tour" flag from the server forever into the future.

And you're right it no longer appears to work with this patch applied.

Reply:
Unfortunate! I think the persist plugin may not have been migrated to work with the new store registration, so that may require more work before we can migrate this one.

Thanks for taking a look!

Reply:
for context, the persistence plugin definitely isn't compatible with the more modern registration methods :( See https://github.com/WordPress/gutenberg/issues/11449

Another question is whether we could use the editor preferences layer instead, which does support browser persistence